eslint.config.js

```diff
@@ -156,10 +174,27 @@ module.exports = [
         }
     },
 
+    // JSON files
+    {
+        name: "eslint/json",
+        files: ["**/*.json", ".c8rc"],
+        ignores: ["**/package-lock.json"],
+        language: "json/json",
+        ...json.configs.recommended
+    },
+
```

Comment:
In the future we could consider putting this into 'eslint-config-eslint' so that it can be easily used in other repositories of the eslint organization.